src/it/MPMD-335-aggregate-classpath-repositories/verify.groovy

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+File buildLog = new File( basedir, 'build.log' )
+assert buildLog.exists()
+assert !buildLog.text.contains( 'The POM for org.apache.maven.plugins.pmd.it:mpmd335-dependency1:jar:1.0.0-SNAPSHOT is missing, no dependency information available' )
+assert !buildLog.text.contains( 'org.eclipse.aether.resolution.DependencyResolutionException: Could not find artifact org.apache.maven.plugins.pmd.it:mpmd335-dependency1:jar:1.0.0-SNAPSHOT' )
```

Comment:
This verifies actually a wrong log message. the test dependency is now package pom and not jar anymore...